.internal-ci/charts/reserve-auditor/templates/statefulset.yaml

```diff
@@ -72,9 +64,9 @@ spec:
             failureThreshold: 4
             periodSeconds: 30
           volumeMounts:
-            - name: {{ $.Chart.Name }}-ledger
+            - name: {{ $.Chart.Name }}
```

Comment:
I don't think you're going to be able to reuse the same volume `{{ $.Chart.Name }}` here like this.  You probably need to rewire the container/app to put `/ledger` and `/mobilecoind` under a single path like `/data`  

So you end up with a single volume  mount at `/data`  and sub directories created by the app/entrypoint for `/data/ledger`, `/data/mobilecoind` ...

Reply:
There are a few other places that might need the same sub directory patterns.

Reply:
Well I stand corrected,  I guess that will work within a pod.